modules/nw-ingress-sharding-namespace-labels.adoc

```diff
@@ -15,6 +15,11 @@ a set of Ingress Controllers and when isolating traffic to a specific Ingress
 Controller. For example, company A goes to one Ingress Controller and company B
 to another.
 
+[WARNING]
+====
+If you deploy the Keeplived Ingress VIP, do not deploy a non-default Ingress Controller with value `HostNetwork` for the `endpointPublishingStrategy` parameter. Doing so might cause issues. Use value `NodePort` instead of `HostNetwork` for `endpointPublishingStrategy`.
```

Comment:
One tiny nit and otherwise this LGTM:
s/Keeplived/Keepalived

Reply:
done